src/components/site-logo/index.tsx

```diff
@@ -0,0 +1,29 @@
+import type { QwikIntrinsicElements } from "@builder.io/qwik";
+import { component$ } from "@builder.io/qwik";
+
+import styles from "./site-logo.module.css";
+
+export const SiteLogo = component$((props: QwikIntrinsicElements["span"]) => {
```

Comment:
I'm doing that same prop-shadowing thing here where if someone wants to pass props to the site logo (e.g. to change its size, modify the color, whatever) they can pass it in to `SiteLogo` rather than having to wrap it in something else.